gui/gui-manager.cpp

```diff
@@ -517,7 +517,7 @@ void GuiManager::processEvent(const Common::Event &event, Dialog *const activeDi
 	Common::Point mouse(event.mouse.x - activeDialog->_x, event.mouse.y - activeDialog->_y);
 	switch (event.type) {
 	case Common::EVENT_KEYDOWN:
-		activeDialog->handleKeyDown(event.kbd);
+		activeDialog->handleKeyDown(event.kbd, event.synthetic);
```

Comment:
Yeah I was wondering about that. Other than the event recorder, are there any other way to generate synthetic events that are not repeat key down events? And do you know of a better way to detect repeat key down events? Your suggestion to just add a TODO suggests you do not, but I though I might ask anyway ;)